server/eb/Makefile

```diff
@@ -20,6 +20,9 @@ build: clean
 	cp app.py artifact.dir/application.py; \
 	cp ../requirements.txt artifact.dir; \
 	cp -r .ebextensions artifact.dir; \
+	if [ -f config.yaml ] ; then \
```

Comment:
something that would be useful down the road (ie, maybe file as an enhancement ticket if you agree) is ability to separately deploy the config from the code.

To do that, I would imagine introducing an environment variable such as CXG_CONFIG, that points at a data location containing the YAML.

Definitely for later - just wanted to drop the idea in here

Reply:
Yes, agreed.  I wasn't completely happy with this solution.